test/integration/multikueue/suite_test.go

```diff
@@ -96,7 +96,11 @@ func createCluster(setupFnc framework.ManagerSetup, apiFeatureGates ...string) c
 		APIServerFeatureGates: apiFeatureGates,
 	}
 	c.cfg = c.fwk.Init()
-	c.ctx, c.client = c.fwk.RunManager(c.cfg, setupFnc)
+	c.ctx, c.client = c.fwk.SetupClient(c.cfg)
+	// skip the manager setup if setup func is not provided
+	if setupFnc != nil {
```

Comment:
Is this needed for the PR?

Reply:
Yes , the worker clusters are fully initialized (including the managers) in before suite, the manager's manager is started/stopped per spec.

Reply:
ok